test/unit/features/directives/bind.spec.js

```diff
@@ -121,6 +121,18 @@ describe('Directive v-bind', () => {
     expect(vm.$el.children[1].innerHTML).toBe('<span>qux</span>')
   })
 
+  it('.prop modifier with normal attribute bind', () => {
+    const vm = new Vue({
+      template: '<input :some.prop="some" :id="id">',
+      data: {
+        some: 'hello',
+        id: 'world'
```

Comment:
Actually, the test case should use a null or false value and check that the id attribute is absent on the element